src/components/App.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import NavBar from './navigation/NavBar';
+import Main from './editor/Main';
+import Editor from './editor/Editor';
+
+const mapStateToProps = (state) => {
+  const { pageId, sectionId } = state.location.payload;
```

Comment:
It would probably be a good idea to write some selectors for redux-first-router. `getSelectedPage` and `getSelectedSection` are easier to use and more consistent with the other selectors.